frontend/tests/features/homepage.test.ts

```diff
@@ -1,64 +1,15 @@
-import { PROMPT_TEXT } from "src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/Details";
-import { goToPage, tryUntil } from "tests/utils/helpers";
-import { getTestTag, getText } from "../utils/selectors";
-
-const DATASET_ROW_DOWNLOAD_BUTTON_ID = "dataset-download-button";
+import { goToPage } from "tests/utils/helpers";
+import { getTestClass, getTestID } from "../utils/selectors";
 
 describe("Homepage", () => {
   it("renders the expected elements", async () => {
     await goToPage();
 
-    await expect(page).toHaveSelector(getText("cellxgene@chanzuckerberg.com"));
-    await expect(page).toHaveSelector(getTestTag("logo"));
-    await expect(page).toHaveSelector(getText("Dataset name"));
-    await expect(page).toHaveSelector(getText("View in cellxgene"));
-    await expect(page).toHaveSelector(getText("Download dataset"));
-    await expect(page).toHaveSelector(getText("More information"));
-
-    await expect(page).toHaveSelector(getTestTag("dataset-name"));
-    await expect(page).toHaveSelector(getTestTag("view-dataset-link"));
-    await expect(page).toHaveSelector(
-      getTestTag(DATASET_ROW_DOWNLOAD_BUTTON_ID)
-    );
-  });
-
-  describe("renders the download dataset modal", () => {
```

Comment:
no longer showing download modal on homepage

Reply:
Awesome thank you!